### DISCUSSION TITLE
Validate Users?

### DISCUSSION THREAD

**Discussion by KarenDouglas** [ykdojo/defaang#26] [General]
Will users sign up to submit/read  questions?

**Comment by ykdojo** [comment]
Reading questions, no. I think it should be open.

Submitting questions, initially, yes. Even when people choose to stay anonymous, I think it’s important that we record their email addresses so we can ask them to revise their questions if they’re not sufficiently clear for us to show on the website. Later, though, there should be an option to “un-associate” the questions from your emails.
